scripts/earn_locked_cvx.py

```diff
@@ -30,15 +30,18 @@
         base_oracle_address=MULTICHAIN_CONFIG.get(chain).get("gas_oracle"),
     )
 
-    # strategy = web3.eth.contract(
-    #     address=ETH_BVECVX_STRATEGY, abi=get_abi(chain, "strategy")
-    # )
-    # vault = web3.eth.contract(address=ETH_BVECVX_VAULT, abi=get_abi(chain, "vault"))
+    strategy = web3.eth.contract(
+        address=ETH_BVECVX_STRATEGY, abi=get_abi(chain, "strategy")
+    )
+    vault = web3.eth.contract(address=ETH_BVECVX_VAULT, abi=get_abi(chain, "vault"))
 
-    # bvecvx_strat_name = "Badger Vested Escrow Convex Token"
+    bvecvx_strat_name = "Badger Vested Escrow Convex Token"
 
-    # logger.info(f"+-----Earning {bvecvx_strat_name}-----+")
-    # earner.earn(vault, strategy, bvecvx_strat_name)
+    logger.info(f"+-----Earning {bvecvx_strat_name}-----+")
+    try:
+        earner.earn(vault, strategy, bvecvx_strat_name)
+    except Exception:
+        logger.error("ERROR EARNING BVECVX")
```

Comment:
Maybe you can also log the error reason

Reply:
underlying class shoudl this is just to handle gracefully so we can still earn graviaura